Vertexvis/vertex-web-sdk
# [SDK-637] Support connecting to FSS

## What

Splits our viewer SDK into two separate packages, being `viewer` (for the platform and connecting to FSS) and `viewer-eedc` (for eedc and connection to ISS). The `viewer` package currently supports loading a scene, but no further interaction, and as such, most of the contents have been removed for now. To be added back:
- View cube
- Viewer toolbar
- Scene manipulation

A note with these changes, all the `viewer-eedc` files have just been moved over from the `viewer` package, [this diff](https://github.com/Vertexvis/vertex-web-sdk/pull/5/files#diff-1868c137ba346e192210526da2d086fcL34) is the start of changes to the `viewer` package that reflect the actual additions in this PR

## Testing 

- Using the `viewer` SDK
  - Provide a scene id through `viewer.load('urn:vertexvis:scene:{id}')`
  - Configure credentials for platdev
  - Verify an image is rendered
